Task_DEV_5/Text1.txt

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
Do we need this file? I don't see places where it is using.

Reply:
Done.